cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,158 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-16880` - Removed schema validation from BQ sink when 'truncate table' option is set.
+
+- :cask-issue:`CDAP-16891` - Unsupported pipelines in drafts would be upgraded when users open them.
+
+- :cask-issue:`CDAP-16940` - UI now waits for 5 mins for inactivity in the browser before stopping all the polling logic.This prevents stopping polling for resources that might take more than 30 seconds to respond (current timeout is 30 seconds).
+
+- :cask-issue:`CDAP-16950` - Includes all ERROR level logs logged under the application logging context.
+
+- :cask-issue:`CDAP-16959` - Fixed an issue with runtime arguments re-rendering and losing focus when containing macros in preview.
+
+- :cask-issue:`CDAP-16972` - Fixed an issue where preview config would open when trying to stop a preview.
+
+- :cask-issue:`CDAP-16975` - UI now adds the latest version of plugin, among the list of different versions of the plugin, when added from the sidepanel in pipeline studio. If the user has already chosen a specific version (older version) it defaults to that instead of the latest.
+
+- :cask-issue:`CDAP-16976` - UI resets the default version of plugins for specific user during upgrade. When users upgrade from 6.1.2 to 6.1.3 or later UI will reset the default version of plugin the user has already chosen. Post upgrade if the user uses the same plugin UI will choose the latest version of the same plugin.
+
+- :cask-issue:`CDAP-16993` - Fixed a bug in preview for fields that have non-string types such as bytes.
+
+- :cask-issue:`CDAP-17000` - Changed default value of spark.network.timeout to 10 minutes to make pipeline execution more stable for shuffle heavy pipelines.
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-16328` - Labeling Dataproc clusters configured as Remote Hadoop Provisioners.
+
+- :cask-issue:`CDAP-16461` - Added Spark parameter to limit Spark block size to prevent issues with joins.
+
+- :cask-issue:`CDAP-16530` - Fixed joiner output schema generation to be deterministic, using the same ordering as they had in the input data.
+
+- :cask-issue:`CDAP-16606` - Limit to reading in 100 records across all input partitions in preview.
+
+- :cask-issue:`CDAP-16621` - Removed modal showing pipeline JSON when users export pipelines. Instead, pipeline gets downloaded when users click "export pipeline" without the extra confirmation step.
+
+- :cask-issue:`CDAP-16656` - Added support for rendering large schemas (>1000 fields) in pipelines UI. By default collapse complex schemas and lazy-load fields in record types.
+
+- :cask-issue:`CDAP-16673` - Added payload compression support to messaging service.
+
+- :cask-issue:`CDAP-16676` - Upgrade to use dataproc API v1beta2 to allow endpoint config.
+
+- :cask-issue:`CDAP-16708` - Added a new AutoJoiner API for plugins to implement. The new API leaves implementation details up to the application, which can perform join optimizations that were not possible with the older Joiner API.
```

Comment:
this should be a new feature, not an improvement.

Reply:
Done